### DISCUSSION TITLE
Site definition

### DISCUSSION THREAD

**Discussion by miekeGR** [ternaustralia/ontology_tern#66] [Modelling]
How should we define a Site i.e. to the level of a re-usable IRI? Thinking from an implementation perspective, we would have to have a mechanism for the Site to be findable for another data submitter to know they are describing the same Site and also from a data user perspective in searching by Site.

**Comment by smguru** [comment]
@miekeGR  I think this is more of an implementation questions, if all the sites have linkeddata.gov.au URI , it can be discoverable from a viewer. Further more, spatial intersect can be checked from existing list.

**Reply by miekeGR** [reply]
Cheers, thanks @smguru , that makes sense.

**Comment by edmondchuc** [comment]
Sites and site visits are usually defined by the survey protocol. See related discussion https://github.com/ternaustralia/ontology_tern/discussions/53#discussioncomment-1776326.
